workspace/notebook/appeal_s78_curated_mipins.json

```diff
@@ -51,6 +51,16 @@
 			"sessionKeepAliveTimeout": 30
 		},
 		"cells": [
+			{
+				"cell_type": "code",
+				"source": [
+					"from pyspark.sql import SparkSession\n",
+					"from notebookutils import mssparkutils\n",
+					"from datetime import datetime, date\n",
+					"from pyspark.sql.functions import col, from_utc_timestamp"
```

Comment:
I would avoid importing `col` like this, since `col` is commonly used as a variable name (not in this notebook but in general)